analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseReturnStatement.vb

```diff
@@ -0,0 +1,82 @@
+﻿Public Class Implicit_return_statements_are_noncompliant
+    Public Function Assigned_return_value_only() As Integer
```

Comment:
Please return this to the old naming style `AssignedReturnValueOnly`. It's easier to read and more authentic to VB conventions.